src/api/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AppConfig(AppConfig):
```

Comment:
i think this should be 
```python
from django.apps import ApiConfig


class ApiConfig(AppConfig)
```
and there will probably be a lot of other naming changes you'll need to make if you haven't done these already: https://stackoverflow.com/a/8408131

Reply:
omg thanks for this link